inception-log/src/main/java/de/tudarmstadt/ukp/inception/log/EventRepository.java

```diff
@@ -17,11 +17,17 @@
  */
 package de.tudarmstadt.ukp.inception.log;
 
+import java.util.List;
+
+import de.tudarmstadt.ukp.clarin.webanno.model.Project;
 import de.tudarmstadt.ukp.inception.log.model.LoggedEvent;
 
 public interface EventRepository
 {
     static final String SERVICE_NAME = "eventRepository";
 
     void create(LoggedEvent aE);
+
+    List<LoggedEvent> listLoggedEvents(Project aProject, String aUsername, String eventType,
```

Comment:
`eventType` and `size` miss the initial `a`.

Reply:
Done. Should be careful